impl-endpoints/cloudflare-go/Dockerfile

```diff
@@ -5,10 +5,10 @@ FROM golang:latest AS builder
 
 RUN apt-get update && \
     apt-get install git
-
-RUN rm -fr /cf-go && git clone --depth 1 --branch cf https://github.com/cloudflare/go /cf-go
+RUN rm -fr /cf-go && git clone --branch cf https://github.com/cloudflare/go /cf-go
```

Comment:
I think we also don't need the branch anymore...

Reply:
Removed it.